Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -7,121 +7,604 @@
 // license that can be found in the LICENSE file.
 
 import UIKit
+import Dcrlibwallet
+import Signals
 
-class OverviewViewController: UIViewController {
-    @IBOutlet weak var syncProgressViewContainer: UIView!
+class OverviewViewController: UIViewController{
+    // Heading & balance
+    @IBOutlet weak var logoImage: UIImageView!
+    @IBOutlet weak var overViewLabel: UILabel! {
+        didSet { self.overViewLabel.text = LocalizedStrings.overview }
```

Comment:
The text property can/should be set from IB using the custom`xibLocalizedStringKey` property. There would be no need to have this outlet here. A number of these `IBOutlets` are actually not necessary. The fonts and localized string key can/should be set from IB.

Reply:
resolved in 7152dc3